VTEX - Master Data API - v2.json

```diff
@@ -120,6 +120,264 @@
                 "deprecated": false
             }
         },
+        "/api/dataentities/CL/documents": {
+            "post": {
+                "tags": ["Customer profiles"],
+                "summary": "Create new customer profile",
+                "description": "Create new customer profile.\r\n\r\n> You can use this request to create customer profiles according to any `CL` schema. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for the schemas you are using. Learn more about how to use [Master Data v2 schemas](https:\/\/developers.vtex.com\/vtex-rest-api\/docs\/master-data-schema-lifecycle).",
```

Comment:
```suggestion
                "description": "Creates new customer profile.\r\n\r\n> You can use this request to create customer profiles according to any `CL` schema. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for the schemas you are using. Learn more about how to use [Master Data v2 schemas](https:\/\/developers.vtex.com\/vtex-rest-api\/docs\/master-data-schema-lifecycle).",
```

Reply:
Esse link tá certo? Será que é ele que tá bugando o preview?